Packs/GuardiCore/Integrations/integration-GuardiCore.yml

```
@@ -751,18 +772,21 @@ script:
             params.insecure,
             params.proxy
         );
-
+        console.log('Response from api: ' + JSON.stringify(res))  // print response to console for debugging
         if (res.StatusCode < 200 || res.StatusCode >= 300) {
             if (token) {
-                sendRequest(methodDictionary.logout, urlDictionary.logout, token);
+                request_logout(methodDictionary.logout, urlDictionary.logout, token);
             }
 
             throw 'Request to GuardiCore ' + url + ' failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
         }
 
         if (isRawOutput) {
             return res.Path;
-        } else {
+        } else if(res.Body.length<=0){  // If body is empty return an empty object
```

how will it look for the user in demisto?
```suggestion
        } else if (res.Body.length <= 0) {  // If body is empty return an empty object
```